components/automate-ui/src/app/components/notifications/notifications.component.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('ChefNotificationsComponent', () => {
         ChefNotificationsComponent
       ],
       imports: [
-        StoreModule.forRoot(ngrxReducers, { ...defaultInitialState, runtimeChecks })
+        StoreModule.forRoot(ngrxReducers, { runtimeChecks })
```

Comment:
As mentioned in the other PR, applying spread to the initial state here is wrong... but also it is not even needed, so just removed it in all the files in this commit.